source/documentation/common/prereqs/ref-Host_Firewall_Requirements.adoc

```diff
@@ -5,7 +5,7 @@
 // PPG
 // Install
 
-{enterprise-linux-host-fullname}s and {hypervisor-fullname}s ({hypervisor-shortname}) require a number of ports to be opened to allow network traffic through the system's firewall. The firewall rules are automatically configured by default when adding a new host to the {engine-name}, overwriting any pre-existing firewall configuration. 
+{enterprise-linux-host-fullname}s and {hypervisor-fullname}s ({hypervisor-shortname}) require a number of ports to be opened to allow network traffic through the system's firewall. The firewall rules are automatically configured by default when adding a new host to the {engine-name}, overwriting any pre-existing firewall configuration.
 
```

Comment:
Good change, but related to this PR - it is best to submit such cleanups separately.